pkg/scaffold/v2/crd/enablewebhook_patch.go

```diff
@@ -68,7 +66,7 @@ spec:
       # but we're going to set it later using the cert-manager (or potentially a patch if not using cert-manager)
       caBundle: Cg==
       service:
-        namespace: $(NAMESPACE)
+        namespace: system
         name: webhook-service
         path: /convert-{{ lower .Resource.Kind }}
```

Comment:
I think we enable conversion webhook at the default path `/convert`, so we need to take the kind out.

Reply:
Good catch.